assignment-2/7s.md

```diff
@@ -1,2 +1,91 @@
+* *Principals* .- (think about measures that will improve performance).
+One of the most remarcable principals is that these seven elements need to be aligned and helping each other in order to hold performance at the host organization, and organizations must be align with external enviroment. 
+The model can help to identify non aligned issues at critical moments like during changes at the organization. Also to clarify some of the key issues that must be addressed for such organizations to be effective and adaptive.
+
+* *Objectives* .- **Strategy** : What makes the organization special?. Ones such question is answer of course based on the organization vison statement. The next step is to identify the kind of problems the organization addresses on its functional areas, aiming to enhance its competitive advantage. **Structure**: How the organization is arranged or configured? For example some organizations are divided for their geographical distribution, others for its functionalities, others focussed on their main production, aiming to clarify its integration and coordination mechanisms.
+**Systems**:  It refers to the formal procedures for measurement, rules and processes for getting things done and managing the activity of the organization. They include personnel evaluation , rewards, performance assessments, chain-ofcommand relationships, policies for planning, monitoring, and executing projects.
+**Staffing**: It refers to organization's human resources and how they are recruited, selected, and trained to perform their jobs, aiming to choose what is best for its demographic, educational and attitudinal characteristics.
+**Skills**:  What is the organization good at, and what special skills and competencies are found its people, and at the organization itself?. Aiming to identify organization's core competencies and distinctive capabilities.
+**Style**: How do the leaders interface with subordinates and others in the organization, and how do members interact with each other?. Aiming to identify typical behavior patterns of key groups.
+**Shared values**: Are the underlying principles or ideas that provide meaning and purpose in an organization.
+
+* *Limitations* .-
+Is there a recepy to follow for example when something wronge is identify?
+
+## 2. Through the eyes of 7S model.
+
+* **2.1 Software industry:**
+
+### Spotify
+
+Spotify is a commercial music streaming, podcast and video service, founded in april 2006 by Daniel Ek, Martin Lorentzon. Spotify was launched in September 2008 and by June 2015 Spotify had more than 75 million active users, including about 20 million paid users. 
+
+##### Strategy
+
+In order to answer the question, What make spotify special?, we could say that they are revolutionizing music industry, making it available with a simple app at your mobil phone or many other electronic devices.
```

Comment:
The question should be quoted or in italics. Mobile phone is missing an e. Although it is true that they were the first company offering music on streaming for "free", saying that they are "revolutionising" the market is a powerful statement that without support or additional information it sounds like a standard commercial.